### ISSUE TITLE
comparison/survival y-axis is in the wrong place

### ISSUE THREAD

**Issue by tmazor** [cBioPortal/cbioportal#8374]
user reported bug: https://groups.google.com/u/1/g/cbioportal/c/rBTmQkxn76U

y-axis shows up at x=10 rather than x=0 for disease-free survival at link below

![image](https://user-images.githubusercontent.com/10508276/109164881-eba8f000-7748-11eb-86bd-e5dad7fb046d.png)

https://www.cbioportal.org/results/comparison?genetic_profile_ids_PROFILE_MUTATION_EXTENDED=kirc_tcga_mutations&genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION=kirc_tcga_gistic&cancer_study_list=kirc_tcga&Z_SCORE_THRESHOLD=2.0&RPPA_SCORE_THRESHOLD=2.0&data_priority=0&profileFilter=0&case_set_id=kirc_tcga_cnaseq&gene_list=ADAMTS10%250A%2520%250ANLRP2%2520%250A%250AOBSCN%250A%2520%250ASECISBP2L%250A%2520%250AUSP40&geneset_list=%

**Comment by dippindots** [comment]
@tmazor @jjgao I found out this issue is a data issue, in some cases, we have data that has `negative months (e.g. -12.2 months)`, is it a valid data point? Should we fix this in data level?

**Comment by inodb** [comment]
Yup looks like it's `TCGA-CJ-4641`: https://media.githubusercontent.com/media/cBioPortal/datahub/master/public/kirc_tcga/data_bcr_clinical_data_patient.txt
